build/room_ensurer/room_ensurer.py

```diff
@@ -369,7 +374,7 @@ def _connect(self, server_name: str, server_url: str) -> Optional[Tuple[str, GMa
         password = self._password
 
         if server_name != self._own_server_name:
-            signer = make_signer()
+            signer = LocalSigner(hashlib.sha256(self._password.encode()).digest())
```

Comment:
Nitpick: I would only initialise this once in the class constructor and reuse the signer here

Reply:
True, but the number of servers is quite low, so it shouldn't make a difference in practice

Reply:
Agreed, also once an hour is not a too frequent interval to make a difference even for high number of servers